.github/workflows/binaries.yml

```diff
@@ -0,0 +1,126 @@
+name: Binaries Build
```

Comment:
We could take a look at [buildjet,](https://buildjet.com/for-github-actions) to make it easier for us to create ARM64 stuff and test them.

Reply:
I've looked at this option, and I have some reservations due to its cost implications. Since our repository is public, GitHub Actions are free, and our goal is to ensure the build and functionality of the binaries.

Considering the use of [buildjet](https://buildjet.com/for-github-actions) to replace QEMU, we may need to duplicate checks—one using GitHub runners for AMD and another with [buildjet](https://buildjet.com/for-github-actions) for ARM- that would not make things more simple I think. 

The main advantage of [buildjet](https://buildjet.com/for-github-actions) is that it runs on native ARM systems, which could be faster. However, all tests added here don't take more than 5 minutes.

So, the question is whether the cost outweighs the benefits in this case.

Reply:
We have a CI system we've built and are already paying for that excels at building for arm. Speaking from experience with the release ci job - github is fast enough.

Reply:
So, I think we can convey in keep GitHub

Reply:
closing as sorted out.